database/index.js

```diff
@@ -21,85 +21,77 @@ db.query(`CREATE DATABASE IF NOT EXISTS ${database}`, (err, result) => {
     throw err;
   }
   console.log("AirBnB database created!");
-  db.query(useDB, (err, result) => {
-    if (err) {
-      console.log('cannot use');
-      throw err;
-    }
-    console.log("Currently using AirBnB database!");
+});
 
-    const usersTable = `CREATE TABLE IF NOT EXISTS reviews_users (
-      user_id INT AUTO_INCREMENT PRIMARY KEY,
-      user_name VARCHAR(255),
-      user_img INT
-    )`
-    db.query(usersTable, (err, result) => {
-      if (err) {
-        throw err;
-      }
-      console.log('Users table has been created!')
-    })
+db.query(useDB, (err, result) => {
+  if (err) {
+    console.log('cannot use');
+    throw err;
+  }
+  console.log("Currently using AirBnB database!");
+});
 
-    const hostsTable = `CREATE TABLE IF NOT EXISTS reviews_hosts (
-      host_id INT AUTO_INCREMENT PRIMARY KEY,
-      host_name VARCHAR(255),
-      host_img INT
-    )`
-    db.query(hostsTable, (err, result) => {
-      if (err) {
-        throw err;
-      }
-      console.log('Hosts table has been created!')
-    })
+const usersTable = `CREATE TABLE IF NOT EXISTS reviews_users (
+  user_id INT AUTO_INCREMENT PRIMARY KEY,
```

Comment:
What's the purpose of AUTO_INCREMENT when you're assigning user_id manually? If that's the case, could you use zero-based indexing if you nixed AUTO_INCREMENT?